lib/appraisal/command.rb

```diff
@@ -43,8 +43,12 @@ def with_clean_env
     def ensure_bundler_is_available
       unless system %(gem list -q "^bundler$" | grep -q bundler)
         puts ">> Reinstall Bundler into #{ENV["GEM_HOME"]}"
+        version = Gem::Specification.
```

Comment:
Do you think it's worth to just move this to `Utils` class? Maybe `Utils.bundler_version`?